source/guides/network/running-a-symbol-node-manually.rst

```diff
@@ -0,0 +1,194 @@
+
+Run the server
+==============
+
+**Download** the network configuration file from **A LOCATION YET TO BE DETERMINED**.
+
+Follow Catapult server's `instructions to run a node <https://github.com/nemtech/catapult-server/blob/main/docs/RUNPEERLIN.md>`__ using the downloaded network configuration.
+
+*****************
+Build an API node
+*****************
+
+Once you have the Catapult Server running, you can add **extra services** to turn it into an API node:
+
+.. figure:: ../../resources/images/diagrams/rest-detail.png
+    :align: center
+
+As shown in the diagram above, besides the **Catapult Server** you will also need:
+
+- A **Database** holding the REST data.
+- A **Catapult Broker** serializing accesses to the database.
+- A **REST gateway** accepting client requests and turning them into server commands or database queries.
+
+The following sections explain how to install each service. You will need all of them.
+
+Move to the ``_build`` folder created while building the server and create any additional folders requested in the next steps inside ``_build``.
+
+Run the database
+=====================
+
+1. **Install** `MongoDB <http://mongodb.com>`__ (at least version 4.4).
+
+   It is recommended to follow the `program's own installation instructions <https://docs.mongodb.com/manual/administration/install-community/>`__, and NOT install from your distro's packages.
+
+2. Create a directory to store the data files and **run the database**:
+
+   .. code-block:: bash
+
+      mkdir dbfiles
+      mongod --dbpath=dbfiles --wiredTigerCacheSizeGb 2 --bind_ip 127.0.0.1
+
+   Keep ``mongod`` running for as long as your node runs.
+
+3. **Build indices** to optimize database accesses.
+
+   The ``catapult-server`` repository you cloned to build the server in the first step contains `a folder with mongo scripts <https://github.com/nemtech/catapult-server/blob/main/scripts/mongo>`__. Run this one:
+
+   .. code-block:: bash
+
+      mongo 127.0.0.1/catapult < ../../scripts/mongo/mongoDbPrepare.js
+
+   Without this step database performance will be unacceptably slow.
+
+Run the broker
+===================
+
+4. **Edit** ``resources/config-database.properties`` and point ``databaseUri`` to ``127.0.0.1:27017``:
+
+   .. code-block:: ini
+
+      [database]
+  
+      databaseUri = mongodb://127.0.0.1:27017
+      databaseName = catapult
+
+5. **Run the broker**:
+
+   Move into the ``bin`` directory and run:
+
+   .. code-block:: bash
+
+      ./catapult.broker ..
+
+   Keep the broker running for as long as your node runs.
+
+Build and run the REST gateway
+==============================
+
+6. **Clone** the ``catapult-rest`` repository and **install its dependencies**:
+
+   .. code-block:: bash
+
+      git clone https://github.com/nemtech/catapult-rest.git
+      cd catapult-rest
+      npm install -g yarn
+      ./yarn-setup.sh
```

Comment:
typo. the correct form is `./yarn_setup.sh`

Reply:
👍